DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -121,9 +121,9 @@
 #string STR_SLEEP_TYPE_S3     #language en-US "Suspend to RAM (S3)"
 #string STR_SLEEP_TYPE_S3_EXPERIMENTAL     #language en-US "Suspend to RAM (S3) - Experimental"
 
-#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power failure"
-#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply is restored after power failure.\n\n"
-                                                          "Note that the definition of power failure varies between platforms and it might take absence of power for up to 30 seconds for this option to apply."
+#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state when connecting power supply after power failure"
+#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply on desktop platform, and barrel jack / USB-PD charger on laptops is reattached after power failure.\n\n"

```

Comment:
`on desktop platform, and barrel jack / USB-PD charger on laptops`

This applies to all platforms, not only desktops and laptops...

Reply:
@wiktormowinski what about here?